ckanext/dcat_usmetadata/templates/new-metadata.html

```diff
@@ -1,5 +1,5 @@
 <div id="root" data-apiUrl="{{h.full_current_url()}}/../../api/3/action/package_create" data-apiKey="{{c.userobj.apikey}}" data-ownerOrg=123" ></div>
 <link rel="stylesheet" href="/css/main.245a4124.chunk.css" type"text/css">
-<script src="/js/2.d1ac1708.chunk.js"></script>
-<script src="/js/main.6437f873.chunk.js"></script>
+<script src="/js/2.5ed9dd22.chunk.js"></script>
+<script src="/js/main.55fdb757.chunk.js"></script>
```

Comment:
Are these manually updated?

Reply:
Yes, atm -- would love to find a better suggestion

Reply:
@adborden It's possible to remove the hash and pull down the static files from npm. The downside is that browsers are likely to cache the resources, interfering with updates.